### PR TITLE
Feat: entrypoints slider

### PR DESCRIPTION
**Pull request by sorodrigo**
This PR includes the new entrypoints. A slider for medium/large screens and vertical stacked cards for mobile.

### CODE REVIEW

File: frontend/scripts/react-components/home/entrypoints.component.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          'Explore the sustainability of different production regions and identify risks and' +
+          ' opportunities facing downstream buyers.',
+        className: '-end'
+      }
+    ];
+    this.getSliderRef = this.getSliderRef.bind(this);
+    this.onClickPrev = this.onClickPrev.bind(this);
+    this.onClickNext = this.onClickNext.bind(this);
+    this.onResize = debounce(this.onResize.bind(this), 300);
+    this.renderEntrypoints = this.renderEntrypoints.bind(this);
```
Comment (tsubik) [inline frontend/scripts/react-components/home/entrypoints.component.jsx]: we don't actually need those two render functions bindings.

File: frontend/scripts/react-components/home/entrypoints.component.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    this.setState({ isMobile: Entrypoints.getIsMobile() });
+  }
+
+  getSliderRef(el) {
+    this.slider = el;
+  }
+
+  renderEntrypoints(grid) {
+    return this.entrypoints.map(slide => (
+      <div className={grid}>
```
Comment (tsubik) [inline frontend/scripts/react-components/home/entrypoints.component.jsx]: `key` property needed here. It should be 
```
map((slide, index) => (
   <div className={grid} key={index}>
```

File: frontend/scripts/react-components/home/entrypoints.component.jsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import Siema from 'react-siema';
+import Link from 'redux-first-router-link';
+import debounce from 'lodash/debounce';
+
+class Entrypoints extends React.PureComponent {
+  static getIsMobile() {
+    return window.innerWidth <= 700;
```
Comment (tsubik) [inline frontend/scripts/react-components/home/entrypoints.component.jsx]: not sure if that's a correct way, but I'm always using breakpoint foundation small width.

File: frontend/scripts/react-components/home/entrypoints.component.jsx (hunk 1 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/home/entrypoints.component.jsx]: We do cause we’re accessing this.state and this.entrypoints

File: frontend/scripts/react-components/home/entrypoints.component.jsx [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    window.addEventListener('resize', this.onResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.onResize);
+  }
+
+  onClickPrev() {
+    this.slider.prev();
+    this.setState({ currentSlide: this.slider.currentSlide });
```
[the remark this answers is not in the document]
Comment (sorodrigo) [reply frontend/scripts/react-components/home/entrypoints.component.jsx]: True, will change it

File: frontend/scripts/react-components/home/entrypoints.component.jsx (hunk 2 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/home/entrypoints.component.jsx]: 👍🏼

File: frontend/styles/components/homepage/entrypoints.scss [hunk 5]
```diff
@@ -0,0 +1,105 @@
+$entrypoint-slide-height: 550px;
```
[the remark this answers is not in the document]
Comment (sorodrigo) [reply frontend/styles/components/homepage/entrypoints.scss]: You’re right I need to change it somehow

File: frontend/scripts/react-components/home/entrypoints.component.jsx (hunk 3 above)
Comment (sorodrigo) [reply frontend/scripts/react-components/home/entrypoints.component.jsx]: I always try to use that one but in this case the content break before, so added a cuatom one

**Comment by sorodrigo** [comment]
Please review! @tsubik

**Comment by tsubik** [comment]
LGTM :+1:

**Pull request merged**